styles/card2.css

```diff
@@ -0,0 +1,67 @@
+.card2-container {
+  margin: 4rem 1rem;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.card2 {
+  display: inline-block;
+  max-width: 500px;
```

Comment:
you should use rem here, use consistent units throughout

Reply:
here I'm using px to fix the size of the card irrespective of the root.